ovos_config/locations.py

```diff
@@ -12,16 +12,101 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import os
-from os.path import join, dirname, expanduser, exists
+from os.path import join, dirname, expanduser, exists, isfile
 from time import sleep
+from ovos_utils.system import search_mycroft_core_location, is_running_from_module
+from ovos_config.ovos import get_xdg_base, is_using_xdg, get_config_filename, get_ovos_config
```

Comment:
These imports should be moved inside their respective functions to resolve curcular import bug